src/relay/op/memory/memory.cc

```diff
@@ -423,5 +427,24 @@ RELAY_REGISTER_OP("memory.shape_func")
                              return {topi::identity(inputs[0])};
                            });
 
+RELAY_REGISTER_OP("memory.shape_of")
```

Comment:
I think we should call it `vm.shape_of` instead of `memory.shape_of`.

Reply:
yeah, make sense. We probably also want to refactor other instructions.